qr_generator/lib/screens/qr_details/bloc/qr_details_state.dart

```diff
@@ -0,0 +1,48 @@
+part of 'qr_details_bloc.dart';
+
+abstract class QrDetailsState extends Equatable {
+  const QrDetailsState();
```

Comment:
**suggestion**: this should contain the "loaded app" info. The state has to remmember these kind of into and persist those, you shouldn't work on them only in `mapEventToState`